### PR TITLE
Upgrade capi client to 8.10

### PR DESCRIPTION
**Pull request by TBonnin**
## What does this change?

Upgrading capi client to 8.10
Upgrading sbt to 0.13.11
Removing facia-json dependency that is not required anymore
Fixing failing tests after refereshing the test data file
## What is the value of this and can you measure success?

Frontend is throwing out thrift related errors http://kibana.gu-web.net/goto/16cb09613e0643ce1d95e00816f14db4 which should be fixed in recent capi client versions according to @mchv 
## Does this affect other platforms - Amp, Apps, etc?

No
## Request for comment

@johnduffell @rich-nguyen @jamespamplin

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -27,13 +27,12 @@ object Dependencies {
...
   val commonsIo = "commons-io" % "commons-io" % "2.4"
   val cssParser = "net.sourceforge.cssparser" % "cssparser" % "0.9.18"
-  val contentApiClient = "com.gu" %% "content-api-client" % "8.2"
+  val contentApiClient = "com.gu" %% "content-api-client" % "8.10"
   val dfpAxis = "com.google.api-ads" % "dfp-axis" % "2.8.0"
   val dispatch = "net.databinder.dispatch" %% "dispatch-core" % "0.11.3"
   val dispatchTest = "net.databinder.dispatch" %% "dispatch-core" % "0.11.3" % Test
   val exactTargetClient = "com.gu" %% "exact-target-client" % "2.24"
   val faciaFapiScalaClient = "com.gu" %% "fapi-client" % faciaVersion
-  val faciaScalaClient = "com.gu" %% "facia-json" % faciaVersion
```
Comment (TBonnin) [inline project/Dependencies.scala]: `facia-json` is pulled as a dependency of `fapi-client`, so there is not need to add it explicitly. Let me know if I wrongly assume that

File: onward/app/services/repositories.scala [hunk 2]
```diff
@@ -1,11 +1,7 @@
 package services
 
-import com.gu.contentapi.client.model.v1.Content
-import com.gu.contentapi.client.parser.JsonParser
 import common.Edition
 import model.{RelatedContentItem, RelatedContent}
-import org.json4s.native.JsonMethods
-import play.api.libs.ws.WS
```
Comment (TBonnin) [inline onward/app/services/repositories.scala]: Those were unused `imports`

**Comment by cb372** [comment]
I tried upgrading sbt to 0.13.11 locally and it fixed the classpath issue. But I see that it didn't fix it on TeamCity. Weird. Anyway, if disabling cached resolution fixes it then that seems like the best solution for now. I'll keep trying to reproduce the problem so I can file an sbt issue.

**Comment by johnduffell** [comment]
image
😨

**Comment by johnduffell** [comment]
I'll give you a 👍 for the idea, let me know if there's anything specific you want me to check in the code!

**Comment by TBonnin** [comment]
@johnduffell capi client update = update ALL data files 🔨 

The only code change I made (outside of new CAPI type , tests or dependencies update) is there https://github.com/guardian/frontend/pull/12971/files#diff-cddbf1a20ebe144ec9aa35d41a46dbb4R241

File: common/app/model/content.scala [hunk 3]
```diff
@@ -238,7 +238,7 @@ final case class Content(
   )
 
   val twitterProperties = Map(
-    "twitter:app:url:googleplay" -> metadata.webUrl.replace("http", "guardian"),
+    "twitter:app:url:googleplay" -> metadata.webUrl.replaceFirst("^[a-zA-Z]*://", "guardian://"), //replace current scheme with guardian mobile app scheme
```
Comment (TBonnin) [inline common/app/model/content.scala]: The issue was that `https://something` was transformed into `guardians://something` :(

Comment (johnduffell) [inline common/app/model/content.scala]: good work! the old logic was a bit... brittle 🔨

**Comment by johnduffell** [comment]
👍 try it and see

**Pull request merged**